src/pages/guide/components/GuideTooltip/GuideTooltip.tsx

```diff
@@ -67,15 +69,15 @@ const GuideTooltip = () => {
                 <LabelPairedBookCircleQuestionLgRegularIcon
                     className='guide-tooltip__icon guide-tooltip__icon--disabled'
                     data-testid='dt_guide_tooltip_icon'
-                    onClick={() => history.push(GUIDE_URL)}
+                    onClick={() => history.push(GUIDE_URL, { from: currentRoute || 'buy-sell' })}
```

Comment:
the buy-sell is just a super last precaution, but it wont navigate there when user selected language modal or another modal